youtube_dl/extractor/ciscolive.py

```diff
@@ -0,0 +1,148 @@
+            'md5': '993d4cf051f6174059328b1dce8e94bd',
+            'info_dict': {
+                'upload_date': '20180629',
+                'title': 'DevNet Panel-Applying Design Thinking to Building Products in Cisco [DEVNET-1794]',
+                'timestamp': 1530316421,
+                'uploader_id': '5647924234001',
+                'id': '5803751616001',
+                'description': 'md5:291dbd447bf745d1f61d944d9508538f',
+                'location': 'WoS, DevNet Theater',
+                'ext': 'mp4',
+            },
+        },
+        {
+            'url': 'https://ciscolive.cisco.com/on-demand-library/?search.technology=scpsTechnology_applicationDevelopment&search.technology=scpsTechnology_ipv6&search.focus=scpsSessionFocus_troubleshootingTroubleshooting#/',
+            'md5': '80e0c3b87e373fe3a3316b934b8915bf',
+            'info_dict': {
+                'upload_date': '20180629',
+                'title': 'Beating the CCIE Routing & Switching [BRKCCIE-9162]',
+                'timestamp': 1530311842,
+                'uploader_id': '5647924234001',
+                'id': '5803735679001',
+                'description': 'md5:18bf6e8a634df0a51290401f209089b0',
+                'location': 'Tulúm 02',
+                'ext': 'mp4',
+            },
+        }
+    ]
+
+    # These appear to be constant across all Cisco Live presentations
+    # and are not tied to any user session or event
+    RAINFOCUS_API_URL = 'https://events.rainfocus.com/api/%s'
+    RAINFOCUS_APIPROFILEID = 'Na3vqYdAlJFSxhYTYQGuMbpafMqftalz'
+    RAINFOCUS_WIDGETID = 'n6l4Lo05R8fiy3RpUBm447dZN8uNWoye'
+    BRIGHTCOVE_URL_TEMPLATE = 'http://players.brightcove.net/5647924234001/SyK2FdqjM_default/index.html?videoId=%s'
+
+    def _parse_rf_item(self, rf_item):
+        ''' Parses metadata and passes to Brightcove extractor
+
+        '''
+        event_name = rf_item.get('eventName')
+        cl_id = rf_item.get('abbreviation')
+        title = rf_item.get('title')
+        description = clean_html(rf_item.get('abstract'))
+        presenter_name = try_get(rf_item, lambda x: x['participants'][0]['fullName'])
+        presenter_title = try_get(rf_item, lambda x: x['participants'][0]['jobTitle'])
+        pdf_url = try_get(rf_item, lambda x: x['files'][0]['url'])
+        bc_id = rf_item['videos'][0]['url']
+        bc_url = self.BRIGHTCOVE_URL_TEMPLATE % bc_id
+        duration = int_or_none(try_get(rf_item, lambda x: x['times'][0]['length']))
+        location = try_get(rf_item, lambda x: x['times'][0]['room'])
+
+        if duration:
+            duration = duration * 60
+
+        return {
+            '_type': 'url_transparent',
+            'url': bc_url,
+            'id': cl_id,
+            'title': '%s [%s]' % (title, cl_id),
+            'creator': '%s, %s' % (presenter_name, presenter_title),
+            'description': '%s\n\nVideo Player: %s\nSlide Deck: %s' % (description, bc_url, pdf_url),
```

Comment:
Same.

Reply:
If not in the description, is there a more appropriate way to present the slide deck URL?